parcels/field.py

```diff
@@ -976,6 +977,34 @@ def computeTimeChunk(self, data, tindex):
 
         return data
 
+    def __add__(self, field):
+        if isinstance(self, Field) and isinstance(field, Field):
+            return SummedField([self, field])
+        elif isinstance(field, SummedField):
+            return field.__add__(self)
```

Comment:
Here you said that we do support
Field + SummedField
But this will simply compute
SummedField + Field
right? Should we put a warning? Since the indexing is not really used by the users? (maybe?)
Or only support SummedField + Field